styx-flyte-client/src/main/java/com/spotify/styx/flyte/client/FlyteAdminClient.java

```diff
@@ -70,6 +72,14 @@ public static FlyteAdminClient create(String target, boolean insecure) {
             .setNesting(USER_TRIGGERED_EXECUTION_NESTING)
             .build();
 
+    var launchPlan =
+        stub.getLaunchPlan(
```

Comment:
Should we write a getLaunchPlan function in the FlyteAdminClient to be consistent?

Reply:
yes sounds good